KaylaKornelis-app/src/test/java/edu/KaylaKornelis/advancedjava/Assignment1/BookTest.java

```diff
@@ -0,0 +1,60 @@
+package edu.KaylaKornelis.advancedjava.Assignment1;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * JUnit tests for the Book class
+ *
+ * @author Kayla Kornelis
+ */
+public class BookTest {
+    private Book book;
+    private String title;
+    private String author;
+    private String genre;
+
+
+    @Before
+    public void setup() {
+        Book book = new Book("Introduction to Java", "Spencer Marks", "Textbook");
+    }
+
+    @Test
+    public void bookTitleSetCorrectly(){
+        String title = book.getTitle();
+        assertEquals(title, "Introduction to Java");
```

Comment:
you will notice that all the assert methods can take an initial string. 
e.g. assertEquals("verify title is correct", title, "Introduction to Java"); 
Using that version and supplying a diagnostic message makes the test failure messages much easier for you to process. It is a practice I highly recommend.